azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/utilities/BottomCellActionViewHolder.kt

```diff
@@ -0,0 +1,91 @@
+package com.azure.android.communication.ui.utilities
+
+import android.view.View
+import android.widget.ImageView
+import androidx.core.content.ContextCompat
+import androidx.core.view.AccessibilityDelegateCompat
+import androidx.core.view.ViewCompat
+import androidx.core.view.accessibility.AccessibilityNodeInfoCompat
+import com.azure.android.communication.ui.R
+import com.microsoft.fluentui.persona.AvatarView
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+internal class BottomCellActionViewHolder(itemView: View) : BottomCellViewHolder(itemView) {
```

Comment:
Reconsider the name of this class to be less generic, and include documentation on how it would be used.

I think BottomMenuCellActionViewHolder would go a long way to explain what it does. (use Menu) in the names.

"Bottom" by itself isn't really descriptive to what it does.

Reply:
Included in aed9e65